CriticalMass/ChatInputView.swift

```diff
@@ -12,25 +12,29 @@ protocol ChatInputDelegate: AnyObject {
 }
 
 class ChatInputView: UIView, UITextFieldDelegate {
+    struct Constants {
+        static let textFieldHeight = CGFloat(46.0)
```

Comment:
Does that work with dynamic type?

Reply:
That is a good question. I have not tried tbh. Never saw out in the os but I always thought why is no search bar or whatsoever updated with a bigger font. I will take a look in what is possible there

Reply:
![PNG-Bild](https://user-images.githubusercontent.com/7677738/67715744-b6f91d00-f9ca-11e9-84f2-a49872a07b2f.png)
Just checked iMessage and the text input gets updated. But if just seen that we are not supporting it in the current version of the app so its finde that this PR isn‘t changing that. We just should make sure that nothing is broken but it doesn’t need to scale for now

Reply:
Also safari supports it in the url textfield. Maybe we can create an issue for it. Would be cool if we support it